src/Facility.AspNetCore/Facility.AspNetCore.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
I think this has more implications than just adding the framework reference.

Reply:
Maybe it adds a bunch of razor stuff we don't need? I'm not sure. https://docs.microsoft.com/en-us/aspnet/core/razor-pages/web-sdk?view=aspnetcore-6.0

Reply:
It's the recommended target for an app, not a shared library.

Reply:
Actually, maybe not. `Microsoft.NET.Sdk.Razor` is a separate thing.

Reply:
e8f969a31b5eea3d364073467a203db7adda2667 changed.